tests/rbd/rbd_utils.py

```diff
@@ -608,7 +608,7 @@ def initial_rbd_config(**kw):
         kw["config"]["ec-pool-k-m"] = ec_pool_k_m
         rbd_obj.update({"rbd_reppool": rbd_reppool})
 
-    if kw.get("config").get("ec-pool-k-m"):
+    if not kw.get("config").get("skip-ec-pool"):
         if kw.get("config").get("ec_pool_config"):
```

Comment:
As discussed please add default value for ec-pool-k-m while creating RBD object for ecpool.

Reply:
Done.